internal/impl/kafka/franz_reader_ordered.go

```diff
@@ -449,6 +518,7 @@ func (f *FranzReaderOrdered) Connect(ctx context.Context) error {
 			fetches.EachPartition(func(p kgo.FetchTopicPartition) {
 				if len(p.Records) > 0 {
 					batch := f.recordsToBatch(p.Records)
+					// TODO: do we have to ack?
```

Comment:
We had talked earlier and we don't have to ack.

Reply:
Yep, thank you! I forgot to remove the TODO 😁